our_site/planner/views.py

```diff
@@ -204,6 +213,36 @@ def edit_event(request, event_id: int) -> HttpResponse:
     )
 
 
+def event_home_owned(request, event_id):
+    if request.user.is_anonymous:
```

Comment:
There is a utility function event_specific_credentials created so that we don't have to copy and paste this code for checking to see if a user owns an event. It might need to be modified a little to get the Http404 error because at the moment it only does a redirect if they don't own it or aren't logged in, but that way we wouldn't have to write nearly as much code each time you want a function that validates the user in such a way.

Reply:
I guess this isn't a change that needs to be made, just a note so that you don't have to do as much work next time.